game/views.py

```
@@ -520,4 +520,121 @@ def stats_view(request):
+
+    captures = checks = checkmates = promotions = blunders = mistakes = 0
+    move_analysis_details = []
+    game = ChessGame()
+    game.white_time = 10 ** 9
+    game.black_time = 10 ** 9
+    
+    for idx, notation in enumerate(moves[:80]): # Cap at 80 for perf
+        move_num = idx // 2 + 1
+        color = 'White' if idx % 2 == 0 else 'Black'
+
+        if 'x' in notation: captures += 1
```

Ruff (0.15.20)

[error] Multiple statements on one line (colon)

(E701)